tests/test_end_to_end/test_end_to_end.py

```diff
@@ -36,4 +36,9 @@ def test_end_to_end(capsys):
     if result.exit_code != 0:
         raise result.exception
     _compare_directories(gm_path, output_path)
+
+    import mypy.api
+    out, err, status = mypy.api.run([str(output_path), "--strict"])
```

Comment:
Thanks!  While reading the rest of this I was thinking I needed to add something like this.